COPYRIGHT

```diff
@@ -1 +1 @@
-Copyright 2015-2022 Ably Real-time Ltd (ably.com)
+Copyright 2015-2024 Ably Real-time Ltd (ably.com)
```

Comment:
Just a curiosity, I spotted this was 2 years old now, do we need to keep it updated? 👀